src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -685,44 +687,72 @@ function saturation_adjustment_NewtonsMethod(e_int::FT, ρ::FT, q_tot::FT) where
       T -> internal_energy_sat(T, ρ, q_tot) - e_int,
       T_ -> ∂e_int_∂T(T_, e_int, ρ, q_tot),
       T_1,
-      NewtonsMethod(), CompactSolution(), FT(1e-3), 10)
-      if !sol.converged
-        error("saturation_adjustment_NewtonsMethod did not converge")
-      end
+      NewtonsMethod(), CompactSolution(), tol, maxiter)
+    if !sol.converged
+      error("saturation_adjustment did not converge")
+    end
     return sol.root
   end
 end
 
 """
-    saturation_adjustment(e_int, ρ, q_tot)
+    ΔT_min(::Type{FT})
```

Comment:
Do we still need `ΔT_min` and `ΔT_max` with the secant method with the updated (and more physical) tests of saturation adjustment? Otherwise, I'd remove these.

Reply:
I think it makes sense to keep these range bounds, but I can try removing them to see what happens. We may need them again if we increase the number of cases / combinations we test.

Reply:
Looks like this does improve convergence in the standard cases (for the reference profiles). Specifically:
`ts = PhaseEquil.(e_int, ρ, q_tot, FT(1e-2), 15, Ref(MT.saturation_adjustment_SecantMethod))` passes with `T_2 = bound_upper_temperature(T_1, T_2)` in the saturation adjustment function.

Reply:
Ok